indico/modules/events/contributions/templates/management/_contribution_protection_message.html

```diff
@@ -0,0 +1,71 @@
+<div class="form-group" id="form-group-protected">
+    <div class="form-label form-block"></div>
+    <div class="form-field form-block">
+        {% set protection_parent = contrib.protection_parent %}
+
+        <div class="action-box message-only for-form public-protection-message danger">
```

Comment:
public should be green, not red. or do we want a different (new?) color for it to distinguish between inherited-public and always-public?  cc @OmeGak @nop33 @pferreir